h/models/group.py

```diff
@@ -85,11 +84,15 @@ class Group(Base, mixins.Timestamps):
         'User', secondary='user_group', backref=sa.orm.backref(
             'groups', order_by='Group.name'))
 
-    def __init__(self, name, authority, creator, description=None):
+    def __init__(self, name, authority, creator, description=None,
+                 joinable_by=None, readable_by=None, writeable_by=None):
         self.name = name
         self.authority = authority
         self.description = description
         self.creator = creator
+        self.joinable_by = joinable_by
+        self.readable_by = readable_by
+        self.writeable_by = writeable_by
```

Comment:
Yeah, this isn't your fault, but this is getting a bit silly now. We should get rid of the explicit args /kwargs here and use `super`. Perhaps a job for a later commit, but ORM objects have constructors already. Not really sure why this one got overridden.